plugins/inputs/passenger/passenger_test.go

```diff
@@ -0,0 +1,131 @@
+`
+
+func fakePassengerStatus(stat string) {
```

Comment:
I tested by creating a fake shell script to output XML data and set the path of script to `command` field